### DISCUSSION TITLE
Cleanup Reference Voice in XTTS-v2 Huggingface Space

### DISCUSSION THREAD

**Discussion by ukemamaster** [coqui-ai/TTS#3326] [General]
XTTS-v2 has an option of **Cleanup Reference Voice** in the [Huggingface Space](https://huggingface.co/spaces/coqui/xtts). I wonder what does this option do in the code? Can someone point-out the piece of code that cleans the reference speaker wav?

**Comment by erogol** [comment]
code is already there you can see it yourself.

**Comment by DevOps920719** [comment]
I couldn't find that. Could you please tell me where it is?

Thank you.
